pyproject.toml

```
@@ -419,6 +402,17 @@ known-first-party = ["astropy", "extension_helpers"]
 [tool.ruff.lint.pydocstyle]
 convention = "numpy"
 
+[tool.ruff.format]
+exclude = ["examples/*",
+          "docs/*",
+          "docs/wcs/examples/*",
+          "docs/nddata/examples/*",
+          "cextern/*",
+          "astropy/extern/*",
+          "astropy/coordinates/.*tab.py",
+          "astropy/units/format/.*tab.py"]
```

Ruff formatter settings should precede Ruff linter settings to maintain alphabetical ordering. Furthermore, the configuration can be simplified to
```suggestion
[tool.ruff.format]
exclude = ["docs/*", "examples/*"]
```

---

`extern` is already excluded elsewhere, right?

---

https://github.com/astropy/astropy/blob/aeeed0886ebfbc7642042e55920d62a82f3feff7/pyproject.toml#L323-L329